### ISSUE TITLE
[BUG] Ocarina doesn't work? // Horn of the Gods only works in Survival // Any way to disable these items?

### ISSUE THREAD

**Issue by roccscout** [viniciusgf2/Ambience#43]
**Describe the bug**
1) I cannot get the ocarina to work. I am using unobstructed, default keybinds. I have plenty of XP, in survival, I can hold Right click to bring it to my face, and when I use the keys to play any of the songs listed in the wiki... nothing happens. Is this a bug? Is it broken? Is there a special trick to play the ocarina?

2) The Horn of the Gods is really unbalanced for the survival pack I've created. Is there any way to remove this item from the loot pool in chests?

3) The Horn doesn't work in survival. Pressing RMB only makes it "glitch", but it works normally in Creative mode.

**Mod Version used:
-Mod Version **1.7.6**.

**Comment by CalaMariGold** [comment]
Can confirm these points. The sound for the flute is very quiet too and playing two notes back to back immediately stops the previous note.

**Comment by CalaMariGold** [comment]
Just tested the Ocarina in the newest version, still doesn't work. I would still recommend my previous suggestion "playing two notes back to back immediately stops the previous note."

**Comment by roccscout** [comment]
Agreed with CalaMariGold, Ocarina still doesn't work in this version. Horn does work.
